lib/lightning_web/live/project_live/settings.ex

```diff
@@ -0,0 +1,61 @@
+defmodule LightningWeb.ProjectLive.Settings do
+  @moduledoc """
+  Index Liveview for Runs
+  """
+  use LightningWeb, :live_view
+
+  alias Lightning.Projects
+
+  on_mount({LightningWeb.Hooks, :project_scope})
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok,
+     socket
+     |> assign(
+       active_menu_item: :settings,
+       is_user_admin:
+         Projects.get_project_user_role(
+           socket.assigns.current_user,
+           socket.assigns.project
+         ) == :admin,
+       changeset: Projects.change_project(socket.assigns.project)
+     )}
+  end
+
+  @impl true
+  def handle_params(params, _url, socket) do
+    {:noreply, socket |> apply_action(socket.assigns.live_action, params)}
+  end
+
+  defp apply_action(socket, :index, _params) do
+    socket |> assign(:page_title, "Project settings")
+  end
+
+  @impl true
+  def handle_event("validate", %{"project" => project_params} = _params, socket) do
+    changeset =
+      socket.assigns.project
+      |> Projects.change_project(project_params)
+      |> Map.put(:action, :validate)
+
+    {:noreply, assign(socket, :changeset, changeset)}
+  end
+
+  def handle_event("save", %{"project" => project_params} = _params, socket) do
+    save_project(socket, project_params)
```

Comment:
I'm a little concerned that we are relying solely on disabled fields to protect changes to the project. I didn't manage to easily break this, but I think with a well crafted LiveSocket event in Javascript on could easily save params that aren't in the form.

I don't think this branch is the place to address this, but we should have a discussion about approaches that we can take to lock this stuff down